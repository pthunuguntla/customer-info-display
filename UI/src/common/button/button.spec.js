
import { shallow } from 'enzyme';
import renderer from 'react-test-renderer';
import Button from './';

describe('<Button /> with no props', () => {
    it('should match the snapshot', () => {
        const wrapper = shallow(<Button name={"Add"} onClick={jest.fn()} />);
        const tree = renderer.create(wrapper).toJSON();
        expect(tree).toMatchSnapshot();
    });
});