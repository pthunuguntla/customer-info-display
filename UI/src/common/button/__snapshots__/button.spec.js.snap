// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Button /> with no props should match the snapshot 1`] = `
<button
  className="button"
  onClick={[Function]}
>
  Add
</button>
`;
